src/jobs/schedule-run.yml

```diff
@@ -0,0 +1,45 @@
+description: >
+  Clone test project and schedule e2e automated run on Testlio platform.
+
+executor: default
+
+parameters:
+  dir:
+    type: string
+    default: "."
+    description: |
+      Select the path to test project sources. Default is current directory.
+  token:
+    type: string
+    default: "TESTLIO_RUN_API_TOKEN"
+    description: |
+      Enter the name of the environment variable containing the Testlio Run API token to be used for authentication. Default is "TESTLIO_RUN_API_TOKEN".
+  attach-workspace:
+    type: boolean
+    default: false
+    description: |
+      Boolean for whether or not to attach to an existing workspace. Default is false.
+  test-config:
+    type: string
+    default: "./test-config.json"
+    description: |
+      Specify path to the test config JSON file. Default is "./test-config.json".
+  project-config:
+    type: string
+    default: "./project-config.json"
+    description: |
+      Specify path to the project config JSON file. Default is "./project-config.json".
+steps:
+  - when:
+      condition: << parameters.attach-workspace >>
+      steps:
+        - attach_workspace:
+            at: ./
+  - install
+  - package:
```

Comment:
As I mentioned in this comment https://github.com/Testlio/testlio-circleci-orb/pull/1#discussion_r893146680, for the the job of the Testlio CLI, and the CircleCI orb is to create a run, assuming that zip file with test scripts and build are available somewhere in the filesystem.

The packaging should not be responsibility of the Testlio CLI or the CircleCI orb.